lib/question.js

```diff
@@ -111,6 +111,13 @@ module.exports = {
                     options: data.options || {},
                     broken: hasFatalIssue,
                 };
+                if (question.workspace_image !== null) { // if workspace, add graded files to params
+                    variant.params['_workspace_required_file_names'] = question.workspace_graded_files;
```

Comment:
When possible, we should always assume that data from DB might be `NULL`, unless we have an explicit non-NULL constraint. In this case we don't, so we should treat `question.workspace_graded_files` as potentially NULL and do something like:
```suggestion
                    variant.params['_workspace_required_file_names'] = question.workspace_graded_files || [];
```

Reply:
I see, so even though it's required by the JSON schema, something still might have gone wrong in the JSON sync?